ASWF/meetings/tsc/2020-11-30.md

```diff
@@ -0,0 +1,212 @@
+    - Doug: As config working group moves forward a lot of the same questions 
+      are going to come up. A lot of questions from initial config discussion 
+      was on things like naming and interface.
+    - **TODO**: Michael will help organize a UX working group. 
+
+* Categories discussion (Issue #1222):
+    - Doug: At last TSC meeting talked about this. Created issue #1222. Trying 
+      to collect a number of new features and discussions on how color spaces 
+      appear in application menus. Motivation is the new categories attribute. 
+      Hadn't gotten to the list of proposed categories, or sorted out how we 
+      expect applications to use this. Want to make sure we're all on the same 
+      page in case tweaks are needed to get this working as expected. Proposed 
+      some tweaks in the issue. One of the things that came up in v2 working 
+      group meetings was request from people that they want to specify, from 
+      studio perspective, env vars to filter color spaces for artist 
+      groups, like compositors, lighting artists, modelers, etc. Different from 
+      app developer point of view with different menus for tasks. Proposed both 
+      use cases and something which could work on both ends. Want feedback on 
+      whether I've captured this correctly. See issue content:
+      https://github.com/AcademySoftwareFoundation/OpenColorIO/issues/1222
+    - Kevin: Have existing concept of roles. Fixed set of assumed roles plus 
+      extended set. How would we choose to filter those with this. Nuke now 
+      shows roles in menus, and would be good to filter those as well.
+    - Doug: App helpers code has setting to show roles or not.
+    - Kevin: Want users to find the roles they want to find. Artists trained 
+      to use particular roles for certain tasks.
+    - Deke: Mari has roles listed too, and has its own roles.
+    - Mark B: Intention for roles was just to make sure basic color spaces were 
+      there. Roles have been misused since then. Don't think roles should 
+      appear at all in menus.
+    - Deke: Like roles there, but need to filter them.
+    - Mark B: Category system is powerful. If DCCs have prefs as to what 
+      category might be associated with a dropdown (i.e. texture in Maya). DCCs 
+      can assign categories to UI elements and only show those. Don't think 
+      this should be OCIO-wide env var. Should be per-DCC.
+    - Doug: Yes, would be fixed for certain menu. Specific documented category 
+      so config authors know what an app uses.
+    - Mark B: Could end up with overlapping categories. If up to user they 
+      could define that themselves.
+    - Doug: Proposing standard categories to encourage consistency, but doesn't 
+      need to be limited to this. The key part is there is a set pre-defined to 
+      an application. So Maya would say what categories it's using for different 
+      menus. Would expect to find these at runtime.
```

Comment:
Would expect to find these at runtime. --> There is another set that is end-user defined and would expect to find those at runtime.

Reply:
Fixed.